[fix] Correct the content and Update the angle bracket linking in glossary page and correct link formatting

Vandit1604 opened the pull request:
Fixes #83 and  #80  
with Removed usage of angle brackets in glossary and some pages

Vandit1604 commented:
I'm using this PR for both the issues. The effort is to remove the usage of angle brackets linking as they make the links navigate to top of the page if the resource id isn't found and doesn't raise any error in antora logs.

krisstern commented:
> I'm using this PR for both the issues. The effort is to remove the usage of angle brackets linking as they make the links navigate to top of the page if the resource id isn't found and doesn't raise any error in Antora logs.

I don't understand the context: why only part of #80 is resolved while you claim here both issues?

krisstern commented on docs/projects/modules/gsoc/pages/2021/project-ideas/remoting-monitoring.adoc:
```diff
@@ -1,17 +1,34 @@
+<iframe src="https://docs.google.com/document/d/1nhb-lfdjxT2fAY3US15HIMvePQCCoWU82VV1Y8Mu3lc" width="100%" height="600px"></iframe>
+++++
 
-# To be confirmed
-#- "linuxsuren"
-#- "mikecirioli"
-#- "markyjackson-taulia"
```
Why are these deleted?

Vandit1604 replied:
These are comments. If we want them. I can add them back.

krisstern replied:
These are not just comments, instead they are comments of historical significance. Please add these back.

Vandit1604 commented:
> I don't understand the context: why only part of #80 is resolved while you claim here both issues?

When I raised this PR, I fixed the angle brackets links that were in glossary page which are in this [issue](https://github.com/Vandit1604/jenkins-docs/issues/80#issue-2044430868) submitted by kevin.

With this commit I have fixed the rest of link navigation.
Also I have commented some things on that issue related to some links 
See here https://github.com/Vandit1604/jenkins-docs/issues/80#issuecomment-1858939768

krisstern commented:
> > I don't understand the context: why only part of #80 is resolved while you claim here both issues?
> 
> When I raised this PR, I fixed the angle brackets links that were in glossary page which are in this [issue](https://github.com/Vandit1604/jenkins-docs/issues/80#issue-2044430868) submitted by kevin.
> 
> With this commit I have fixed the rest of link navigation. Also I have commented some things on that issue related to some angle links See here [#80 (comment)](https://github.com/Vandit1604/jenkins-docs/issues/80#issuecomment-1858939768)

Now I see the logic! Thanks for clarifying @Vandit1604
